scripts/modules/createFeed.mjs

```diff
@@ -15,14 +15,15 @@ export default function createFeed(feed, counter){
 
   if(index === 0){
     feedContainer.innerHTML = ""
-  };  
+  };
+  if(loaded > feed.length){loaded = feed.length}  
```

Comment:
This was my fix for the problem. The reason i did it this way to start with was to have a "floating" counter to load 20 posts at a time using the "load more" button to increment the loaded variable